dask/utils.py

```diff
@@ -486,12 +486,16 @@ def ignore_warning(doc, cls, name):
         # Indentation of next line
         indent = re.match(r'\s*', tail).group(0)
         # Insert the warning, indented, with a blank line before and after
-        doc = ''.join([
+        bits = [
             head,
             indent, l1,
             indent, l2, '\n\n',
             tail
-        ])
+        ]
+        if extra:
+            bits.insert(1, extra.rstrip('\n') + '\n\n')
+            bits.insert(1, indent)
```

Comment:
Would it be possible to move extra below the "this is copied" disclaimer?

```
Signature: dd.DataFrame.describe(self, split_every=False, percentiles=None, percentiles_method='default')
Docstring:
Generate descriptive statistics that summarize the central tendency,
dispersion and shape of a dataset's distribution, excluding
``NaN`` values.

Currently, only numeric describe is supported

This docstring was copied from pandas.core.frame.DataFrame.describe.

Some inconsistencies with the Dask version may exist.

Analyzes both numeric and object series, as well
as ``DataFrame`` column sets of mixed data types. The output
will vary depending on what is provided. Refer to the notes
below for more detail.
```

Reply:
done